client/src/components/RunPipelineMenu.vue

```diff
@@ -0,0 +1,136 @@
+<script>
+import { runPipeline } from '@/common/viame.service';
+import { mapActions } from 'vuex';
+
+export default {
+  props: {
+    selected: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  computed: {
+    selectedEligibleClips() {
+      return this.selected.filter(
+        ({ _modelType, meta }) => _modelType === 'folder' && meta && meta.annotate,
+      );
+    },
+    pipelinesNotRunnable() {
+      return this.selectedEligibleClips.length < 1 || !this.pipelines;
+    },
+  },
+
+  asyncComputed: {
+    pipelines() {
+      return this.fetchPipelines();
+    },
+  },
+
+  created() {
+    this.fetchPipelines();
+  },
+
+  methods: {
+    ...mapActions(['fetchPipelines']),
+
+    async runPipelineOnSelectedItem(pipeline) {
+      const clips = this.selectedEligibleClips;
+      await Promise.all(
+        this.selectedEligibleClips.map((item) => runPipeline(item._id, pipeline)),
+      );
+      this.$snackbar({
+        text: `Started pipeline on ${clips.length} clip${
+          clips.length ? 's' : ''
+        }`,
+        timeout: 6000,
+        immediate: true,
+        button: 'View',
+        callback: () => {
+          this.$router.push('/jobs');
```

Comment:
I think it's generally better to use direct component names over paths:

```suggestion
          this.$router.push({ name: 'jobs' });
```